### ISSUE TITLE
Installing protagonist 2.0.0-pre.3 fails with missing source files

### ISSUE THREAD

**Issue by kylef** [apiaryio/protagonist#224]
```
$ npm install protagonist --save

> protagonist@2.0.0-pre.3 install /Users/kyle/Projects/apiaryio/api-elements.js/packages/fury-adapter-oas3-parser/node_modules/protagonist
> node-gyp rebuild

  CXX(target) Release/obj.target/libdrafter/drafter/src/drafter.o
In file included from ../drafter/src/drafter.cc:10:
In file included from ../drafter/ext/snowcrash/src/snowcrash.h:12:
In file included from ../drafter/ext/snowcrash/src/BlueprintSourcemap.h:12:
In file included from ../drafter/ext/snowcrash/src/Blueprint.h:17:
In file included from ../drafter/ext/snowcrash/src/MSON.h:19:
../drafter/ext/snowcrash/ext/markdown-parser/src/MarkdownParser.h:14:10: fatal error: 'markdown.h' file not found
#include "markdown.h"
         ^~~~~~~~~~~~
1 error generated.
make: *** [Release/obj.target/libdrafter/drafter/src/drafter.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/kyle/.config/fnm/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at emitTwo (events.js:126:13)
gyp ERR! stack     at ChildProcess.emit (events.js:214:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
gyp ERR! System Darwin 18.2.0
gyp ERR! command "/Users/kyle/.config/fnm/bin/node" "/Users/kyle/.config/fnm/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/kyle/Projects/apiaryio/api-elements.js/packages/fury-adapter-oas3-parser/node_modules/protagonist
gyp ERR! node -v v8.11.3
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
npm WARN fury-adapter-oas3-parser@0.3.0 requires a peer of fury@3.0.0-beta.8 but none is installed. You must install peer dependencies yourself.
```

**Comment by kylef** [comment]
This was caused by improper release of Protagonist where the git submodules was not synced correctly, we should make a new release with no code changes to resolve.

**Comment by pksunkara** [comment]
Fixed with v2.0.0-pre.4
